instagram-page-react/src/pages/profilePage/organisms/posts/reducers/actionTypes/index.js

```diff
@@ -0,0 +1,7 @@
+const SET_POST = 'setPost';
```

Comment:
This file should be moved to `pages/profilePage/reducers/profilePage.actionTypes`